Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+# To initate the build, run
+# `docker build -t "pimeet" .`
+# from the command line.
+
+WORKDIR /root/build
+  
+RUN  apt-get update \
+  && apt-get install -y wget unzip whois git tree sudo openssh-client \
+  && rm -rf /var/lib/apt/lists/*
+
+ADD ./build /root/build
+ADD ./scripts /root/scripts
+
+CMD   ssh-keygen \
```

Comment:
this would leave your newly-generated SSH creds on the build container, so they may not be accessible after you actually build this. Would it be better to try and find / mount any existing SSH creds to the build container, rather than generating new ones?

Reply:
that will work as well but may be under a different user scope.

Reply:
I'm honestly not too sure, since it's been awhile since I've used this particular image.